### PR TITLE
Multival import

### PR DESCRIPTION
**Pull request by byw**

### CODE REVIEW

File: test/functional/importer_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    create_custom_fields!(@issue)
+    create_versions!(@project)
+    User.stubs(:current).returns(@user)
+  end
+
+  test 'should handle multi-value fields' do
+    post :result, build_params
+    assert_response :success
+    @issue.reload
+    assert_issue_has_affected_versions(@issue, ['Admin', '2013-09-25'])
```
Comment (phillmv) [inline test/functional/importer_controller_test.rb]: 1. just for consistency, should probably assert that the issue doesn't have multi values set before the post?
2. shouldn't you be testing non-version multi values as well?

File: test/functional/importer_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    versions_field = CustomField.find_by_name! 'Affected versions'
+    values = issue.custom_values.find_all_by_custom_field_id versions_field.id
+    assert values.any? {|v| version_ids.include?(v.value) }
+  end
+
+  def create_user!(role, project)
+    user = User.new :admin => true,
+                     :firstname => 'Bob',
+                     :lastname => 'Loblaw',
+                     :mail => 'bob.loblaw@law.blog'
```
Comment (phillmv) [inline test/functional/importer_controller_test.rb]: note on style, I find it BEST PRACTIVE to always use @example.com - you never know when your LOL domain might actually be taken.

File: app/controllers/importer_controller.rb [hunk 3]
```diff
@@ -657,4 +661,15 @@ def version_id_for_name!(project,name,add_versions)
     @version_id_by_name[name]
   end
 
+  def process_multivalue_custom_field(issue, custom_field, csv_val)
+    csv_val.split(',').map(&:strip).map do |val|
+      if custom_field.field_format == 'version'
+        version = Version.find_by_name val
+        version.id
+      else
+        val
```
Comment (phillmv) [inline app/controllers/importer_controller.rb:708]: Are non version custom fields specified by id?

i.e. you split the custom vals, and if they are a version you return the version ids. What if they are not versions?

Comment (byw) [inline app/controllers/importer_controller.rb:708]: It's seems there are only 'version' and 'list' types. And yes, for 'list' types you just return an array of values straight up.

File: test/functional/importer_controller_test.rb (hunk 2 above)
Comment (byw) [inline test/functional/importer_controller_test.rb]: Good point. Should be an easy fix :)

File: test/functional/importer_controller_test.rb (hunk 1 above)
Comment (byw) [inline test/functional/importer_controller_test.rb]: Darn, totally missed the test for list type. Will add that along with some before-post asserts.

**Pull request merged**